datastore/src/main/java/io/spine/server/storage/datastore/DsProperties.java

```diff
@@ -49,6 +60,22 @@ static void addAggregateIdProperty(Object aggregateId, Entity.Builder entity) {
         entity.set(aggregate_id.toString(), propertyValue);
     }
 
+    static void addCreatedProperty(Timestamp when, Entity.Builder entity) {
```

Comment:
The `Property` suffix can be omitted. We have it all under the `DsProperties` class.